src/main/resources/templates/browse.html

```diff
@@ -37,7 +37,7 @@
 <div class="container text-center mt-5">
     <h1>Browse</h1>
     <form class="form-inline justify-content-center mt-3" action="/browse" method="get">
-        <input class="form-control mr-2" type="search" name="query" placeholder="Search" aria-label="Search">
+        <input class="form-control mr-2" type="search" name="query" placeholder="Search" aria-label="Search" autocomplete="on">
         <button class="btn btn-outline-primary" type="submit">Search</button>
```

Comment:
Autocomplete option can be included for the search bar used
to browse movies that could help user save time from typing the complete movie
name.